packages/twenty-front/src/modules/workflow/hooks/useFindWorkflowWithCurrentVersion.tsx

```diff
@@ -0,0 +1,71 @@
+import { CoreObjectNameSingular } from '@/object-metadata/types/CoreObjectNameSingular';
+import { useFindManyRecords } from '@/object-record/hooks/useFindManyRecords';
+import { useFindOneRecord } from '@/object-record/hooks/useFindOneRecord';
+import {
+  Workflow,
+  WorkflowVersion,
+  WorkflowWithCurrentVersion,
+} from '@/workflow/types/Workflow';
+import { isDefined } from 'twenty-ui';
+
+export const useFindWorkflowWithCurrentVersion = (
```

Comment:
rename `useWorkflowWithCurrentVersion`. To be consistent with the rest of the code

Reply:
Done